.travis.yml

```diff
@@ -31,15 +31,21 @@ install:
       bash $MINICONDA_FILE -b
 
       export PATH=/Users/travis/miniconda3/bin:$PATH
-
-      conda config --set show_channel_urls true
-      conda update --yes conda
-      conda install --yes conda-build=1.20.0 jinja2 anaconda-client
       conda config --add channels conda-forge
-      
+      conda config --set show_channel_urls true
+      conda install --yes --quiet conda-forge-build-setup
+      source run_conda_forge_build_setup
 
 script:
   - conda build ./recipe
+  # Test libpng pinnings
+  - conda create -n temp01 --override-channels -c conda-forge -c defaults python=2.7 matplotlib --yes
+  - source activate temp01
+  - python -c 'import matplotlib.pyplot'
+  - conda install --no-update-deps libpng=1.6.21 --yes
+  - python -c 'import matplotlib.pyplot'
+  - conda install --no-update-deps libpng=1.6.23 --yes
+  - python -c 'import matplotlib.pyplot'
```

Comment:
If that implicitly tests whether the png installed by conda is "good", then this should also test that if conda has nothing installed, the tests fail.

I'm not sure which step in https://msdn.microsoft.com/en-us/library/windows/desktop/ms682586%28v=vs.85%29.aspx?f=255&MSPPError=-2147217396 is taken to find the png lib, but my guess is via PATH. If that's correct, then I suspect that having a png lib installed in the system path (step 3. in the `SafeDllSearchMode is disabled` case) will overwrite a png lib installed by conda and therefore that lib (if it the right one) will make the tests succeed :-)

[I know, a bit superficial, but that's why I added the smiley :-)]